ui/packages/consul-ui/app/templates/dc/peers/index.hbs

```diff
@@ -3,59 +3,111 @@
     {{did-insert w.fns.start}}
     {{will-destroy w.fns.stop}}
   </Watcher>
+{{#let
+
+  (hash
+    value=(or sortBy "Name:asc")
+    change=(action (mut sortBy) value="target.selected")
+  )
+
+  (hash
+    searchproperty=(hash
+      value=(if (not-eq searchproperty undefined)
+        (split searchproperty ',')
+        searchProperties
+      )
+      change=(action (mut searchproperty) value="target.selectedItems")
+      default=searchProperties
+    )
+  )
+
+  peers
+
+as |sort filters items|}}
   <AppView>
     <BlockSlot @name="header">
       <h1>
         <route.Title @title="Peers" />
       </h1>
     </BlockSlot>
     <BlockSlot @name="toolbar">
-      <Peerings::Search @search={{this.filter}} @onSearch={{this.handleSearchChanged}} />
+      {{#if (gt items.length 0)}}
+        <Consul::Peer::SearchBar
+          @search={{search}}
+          @onsearch={{action (mut search) value="target.value"}}
```

Comment:
using `pick` and `fn` here would be more idiomatic here in Octane but it's ok to change that in one go for all search related views.